posts/mathlib-changelog.md

```diff
@@ -0,0 +1,55 @@
+---
+author: "David Chanin"
+category: "Community projects"
+date: 2022-07-28 09:35:23 UTC+02:00
+description: ""
+has_math: false
+link: ""
+slug: mathlib-changelog
+tags: ""
+title: Introducing Mathlib Changelog
+type: text
+---
+
+![mathlib-changelog sample page](/images/changelog_lemma.png)
+
+Tldr; check out [mathlib-changelog.org](https://mathlib-changelog.org) to explore the historical changes to mathlib, and find out what happened to that lemma you were using.
+
+<!-- TEASER_END -->
+
+## A changelog for mathlib
```

Comment:
```suggestion
# A changelog for mathlib
```
As the blog machinery automatically ensures this heading is one level lower than the post title.